app/src/main/java/com/nervousfish/nervousfish/MainActivity.java

```diff
@@ -6,8 +6,6 @@
 import android.support.v7.app.AppCompatActivity;
```

Comment:
The changes to the app will cause some issues with #21, I would say that one has priority over this one for now. After #21 is merged, you could add the login frame